lib/YConverter/Converter.php

```diff
@@ -922,7 +922,7 @@ public static function callbackModifyArticleSlices($params)
                 for ($i = 1; $i <= 20; $i++) {
                     $column = 'value' . $i;
                     // Notices bei unserialize vermeiden
-                    if (preg_match('@^a:\d+:{.*?}$@', $slice[$column])) {
+                    if (preg_match('@^a:\d+:{@', $slice[$column])) {
```

Comment:
In der 5er Version (noch nicht auf GH) habe ich hinter dem @ Zeichen `s` Flag notiert.
Wenn das auch bei dir passt, würde ich es gern eher so notieren.

Reply:
Jau, das passt bei mir. Soll ich einen neuen PR machen?

Reply:
Du könntest es in deinem PR direkt auch anpassen

Reply:
check :)

Reply:
Die 5er-Version ist doch auf Github bereits?

Reply:
> Die 5er-Version ist doch auf Github bereits?

Korrekt, ich meinte aber die Anpassung die hier @IngoWinter für die 4er Version gemacht hat.
https://github.com/yakamara/yconverter/blob/6c8b9ec89632f4785f216d25a4d1f19cecf0dc1e/lib/YConverter/Modifier.php#L880